packages/clay-tabs/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import Content from './Content';
+import Item from './Item';
+import React from 'react';
+import TabPane from './TabPane';
+
+interface IProps extends React.HTMLAttributes<HTMLUListElement> {
+	/**
+	 * Justify the nav items according the tab content.
+	 */
+	justified?: boolean;
+
+	/**
+	 * Applies a modern style to the tab.
+	 */
+	modern?: boolean;
+}
+
+export const ClayTabs: React.FunctionComponent<IProps> & {
+	Content: typeof Content;
+	TabPane: typeof TabPane;
+	Item: typeof Item;
+} = ({
+	children,
+	className,
+	justified,
+	modern = true,
+	...otherProps
+}: IProps) => {
+	return (
+		<ul
+			className={classNames(
+				`nav`,
+				{
+					'nav-justified': justified,
+					'nav-tabs': !modern,
+					'nav-underline': modern,
+				},
+				className
+			)}
+			role="tablist"
+			{...otherProps}
```

Comment:
Should this be declared first in the list of props? Otherwise, if someone passes `role` to this item, it'll overwrite `tablist`

Reply:
Oh yeah, thanks!